content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js

```diff
@@ -16,7 +16,8 @@
 (function() {
     "use strict";
 
-    var dataLayer = window.dataLayer = window.dataLayer || [];
+    var dataLayerEnabled = document.querySelector("body").hasAttribute("data-cmp-data-layer-enabled");
```

Comment:
Please use `document.body` instead of `document.querySelector("body")`

Reply:
addressed in https://github.com/adobe/aem-core-wcm-components/pull/991/commits/3abdf0a19dbcaf6c0e167bd846df0bdabe6b24b6